.github/workflows/publish-bulk-cdk.yml

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+name: Publish Bulk CDK
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "airbyte-cdk/bulk"
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: "Repo to check out code from. Defaults to the main airbyte repo."
+        type: choice
+        required: true
+        default: airbytehq/airbyte
+        options:
+          - airbytehq/airbyte
+      build-number:
+        description: "Build Number"
+        required: false
+        type: number
+
+concurrency:
+  group: publish-bulk-cdk
+  cancel-in-progress: false
+
+env:
+  # Use the provided GITREF or default to the branch triggering the workflow.
+  GITREF: ${{ github.event.inputs.gitref || github.ref }}
+  S3_BUILD_CACHE_ACCESS_KEY_ID: ${{ secrets.SELF_RUNNER_AWS_ACCESS_KEY_ID }}
+  S3_BUILD_CACHE_SECRET_KEY: ${{ secrets.SELF_RUNNER_AWS_SECRET_ACCESS_KEY }}
+
+jobs:
+  publish-bulk-cdk:
+    name: Publish Bulk CDK
+    runs-on: connector-test-large
+    timeout-minutes: 30
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.GITREF }}
```

Comment:
I'm not sure this is required. I think that this action checkouts the ref which triggered the workflow be default.

Reply:
Indeed this is detritus I believe. Thanks for noticing!